README.md

```diff
@@ -251,18 +251,18 @@ NOTES:
 * some metric labels are enumerations (`th2_type`: `MESSAGE_GROUP`, `EVENT`, `<customTag>`;`message_type`: `RAW_MESSAGE`, `MESSAGE`)
 
 ABSTRACT METRICS:
-* th2_rabbitmq_message_size_publish_bytes (`th2_pin`, `th2_type`, `exchange`, `routing_key`): quantity of published message bytes to RabbitMQ. The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content
-* th2_rabbitmq_message_publish_total (`th2_pin`, `th2_type`, `exchange`, `routing_key`): quantity of published messages to RabbitMQ. The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content
-* th2_rabbitmq_message_size_subscribe_bytes (`th2_pin`, `th2_type`, `queue`): quantity of received bytes from RabbitMQ, includes bytes of messages dropped after filters. For dropped please see 'th2_message_dropped_subscribe_total' and 'th2_message_group_dropped_subscribe_total'. The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content
-* th2_rabbitmq_message_process_duration_seconds (`th2_pin`, `th2_type`, `queue`): time of message processing during subscribe from RabbitMQ in seconds. The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content
+* th2_rabbitmq_message_size_publish_bytes (`th2_pin`, `th2_type`, `exchange`, `routing_key`): number of published message bytes to RabbitMQ. The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content
+* th2_rabbitmq_message_publish_total (`th2_pin`, `th2_type`, `exchange`, `routing_key`): quantity of published messages to RabbitMQ. The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content
+* th2_rabbitmq_message_size_subscribe_bytes (`th2_pin`, `th2_type`, `queue`): number of bytes received from RabbitMQ, it includes bytes of messages dropped after filters. For dropped message information, please refer to 'th2_message_dropped_subscribe_total' and 'th2_message_group_dropped_subscribe_total'. The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content
```

Comment:
```suggestion
* th2_rabbitmq_message_size_subscribe_bytes (`th2_pin`, `th2_type`, `queue`): number of bytes received from RabbitMQ, it includes bytes of messages dropped after filters. For information about the number of dropped message, please refer to 'th2_message_dropped_subscribe_total' and 'th2_message_group_dropped_subscribe_total'. The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content
```
@jean-michel-perli-sonsin @andreydrobynin what do you think about this option